### PR TITLE
fix: rename novabase to neotalent

### PR DESCRIPTION
**Pull request by afonsojramos**
It seems that Novabase is doing some restructuring and selling its "main" IT business: https://www.linkedin.com/posts/novabase_a-novabase-acaba-de-divulgar-ao-mercado-a-activity-7118339267264692224-rtVP

They have killed the job opportunities from their main site as well.

### CODE REVIEW

**Comment by marmelo** [comment]
Are you saing that Novabase is giving up the IT business?
If not, should we add Neotalent as a new company instead of renaming?

**Comment by vpereira01** [comment]
Hard to parse the information but found it, dropping Novabase makes sense to me. Novabase acts more as a holding company than a real company :D

```
SEGMENT INFORMATION
NOVABASE’s activity is organised into two operating segments: Next-Gen and Value
Portfolio

NEXT-GEN: NOVABASE’s core segment, which operates under the Celfocus commercial
brand according to NOVABASE’s brand architecture. It develops an IT activity with
technological offers that tend to be more advanced and targeted mainly to the Financial
Services and Telecommunications industries and to the most competitive markets (Europe
and the Middle East).

VALUE PORTFOLIO: Segment aggregating the activities of IT Staffing, under the Neotalent
commercial brand, and venture capital through Novabase Capital, S.C.R., S.A.. Its objective
is to generate funds to support Next-Gen strategy. For reporting purposes, Value Portfolio
segment includes the Group’s holding (in 2021, also included the shared services).
```
https://content.novabase.com/storage/uploads/annual-report-2022-nao-esef.pdf

**Comment by afonsojramos** [comment]
It definitely seems like so from everything I've seen so far. The fact that there are no open positions is also a strong indicator. The link I provided also shows that they are clearly pivoting.

**Comment by afonsojramos** [comment]
They've also sold their old govt contract subsidiary.

**Review by vpereira01 — APPROVED** [review]

**Review by marmelo — APPROVED** [review]
